### PR TITLE
New SIP and SRC process

### PR DESCRIPTION
**Pull request by frankTheTank72**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,53 @@
...
+-   **Final** - This SIP represents the final standard. A Final SIP exists in a state of finality and should only be updated to correct errata and add non-normative clarifications.
+-   **Stagnant** - Any SIP in Draft or Review if inactive for a period of 6 months or greater is moved to Stagnant. An EIP may be resurrected from this state by Authors or EIP Editors through moving it back to Draft.
+-   **Withdrawn** - The SIP Author(s) have withdrawn the proposed EIP. This state has finality and can no longer be resurrected using this SIP number. If the idea is pursued at later date it is considered a new proposal.
+-   **Living** - A special status for SIPs that are designed to be continually updated and not reach a state of finality. This includes most notably SIP-Basics.
+
+## SIP Types
+
+SIPs are separated into a number of types:
+
+### Standard Track (45)
```
Comment (ohager) [inline README.md:29]: what does the number mean? I think we can omit it

File: SIP/sip-basic.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+title: SIP Purpose and Guidelines
+status: Living
+type: Informational
+author: jjos, frank_the_tank, ohager
+created: 2022-01-28
+---
+
+## What is an SIP?
+
+SIP stands for Signum Improvement Proposal. An SIP is a design document providing information to the SIgnum community, or describing a new feature for SIgnum or its processes or environment. The SIP should provide a concise technical specification of the feature and a rationale for the feature. The SIP author is responsible for building consensus within the community and documenting dissenting opinions.
```
Comment (ohager) [inline SIP/sip-basic.md:12]:
```suggestion
SIP stands for Signum Improvement Proposal. A SIP is a design document providing information to the Signum community, or describing a new feature for Signum or its processes or environment. The SIP should provide a concise technical specification of the feature and a rationale for the feature. The SIP author is responsible for building consensus within the community and documenting dissenting opinions.
```

File: SIP/sip-basic.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+
+## SIP Types
+
+There are three types of SIP:
+
+- A **Standards Track SIP** describes any change that affects most or all Signum implementations, such as—a change to the network protocol, a change in block or transaction validity rules, proposed application standards/conventions, or any change or addition that affects the interoperability of applications using Signum. Standards Track SIPs consist of three parts—a design document, an implementatio. Furthermore, Standards Track SIPs can be broken down into the following categories:
+- 
+  - **Core**: improvements requiring a consensus fork
+  - **Interface**: includes improvements around client API/RPC. 
+  - **SRC**: application-level standards and conventions, including smart contract standards.
```
Comment (ohager) [inline SIP/sip-basic.md:28]: good :+1:

File: SIP/sip-basic.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+
+It is highly recommended that a single SIP contain a single key proposal or new idea. The more focused the SIP, the more successful it tends to be. A change to one client doesn't require an SIP; a change that affects multiple clients, or defines a standard for multiple apps to use, does.
+
+An SIP must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
+
+### Special requirements for Core SIPs
+
+If a **Core** SIP mentions or proposes changes to the SVM (Signum Virtual Machine), it should refer to the instructions by their mnemonics and define the opcodes of those mnemonics at least once. A preferred way is the following:
+```
+REVERT (0xfe)
```
Comment (ohager) [inline SIP/sip-basic.md:40]: ? what's that?

File: SIP/sip-basic.md [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+
+**Withdrawn** - The SIP Author(s) have withdrawn the proposed SIP. This state has finality and can no longer be resurrected using this SIP number. If the idea is pursued at later date it is considered a new proposal.
+
+**Living** - A special status for SIPs that are designed to be continually updated and not reach a state of finality. This includes most notably SIP-Basic.
+
+## What belongs in a successful SIP?
+
+Each SIP should have the following parts:
+
+- Preamble - RFC 822 style headers containing metadata about the SIP, including the SIP number, a short descriptive title (limited to a maximum of 44 characters), a description (limited to a maximum of 140 characters), and the author details. Irrespective of the category, the title and description should not include SIP number. See [below](./sip-basic.md#sip-header-preamble) for details.
```
Comment (ohager) [inline SIP/sip-basic.md:81]: RFC822? 
https://datatracker.ietf.org/doc/html/rfc822
https://docs.microsoft.com/en-us/previous-versions/office/developer/exchange-server-2010/aa493918(v=exchg.140)

Comment (ohager) [inline SIP/sip-basic.md:81]:
```suggestion
- Preamble - A RFC822 like header containing metadata about the SIP, including the SIP number, a short descriptive title (limited to a maximum of 44 characters), a description (limited to a maximum of 140 characters), and the author details. Irrespective of the category, the title and description should not include SIP number. See [below](./sip-basic.md#sip-header-preamble) for details.
```

File: SIP/sip-basic.md [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+- Reference Implementation - An optional section that contains a reference/example implementation that people can use to assist in understanding or implementing this specification.
+- Security Considerations - All SIPs must contain a section that discusses the security implications/considerations relevant to the proposed change. Include information that might be important for security discussions, surfaces risks and can be used throughout the life-cycle of the proposal. E.g. include security-relevant design decisions, concerns, important discussions, implementation-specific guidance and pitfalls, an outline of threats and risks and how they are being addressed. SIP submissions missing the "Security Considerations" section will be rejected. An SIP cannot proceed to status "Final" without a Security Considerations discussion deemed sufficient by the reviewers.
+- Copyright Waiver - All SIPs must be in the public domain. See the bottom of this SIP for an example copyright waiver.
+
+## EIP Formats and Templates
```
Comment (ohager) [inline SIP/sip-basic.md:92]:
```suggestion
## SIP Formats and Templates
```

**Review by ohager — CHANGES_REQUESTED** [review]
Good job. Please review my suggestions and replace really _all_ EIPs by SIPs

File: README.md (hunk 1 above)
Comment (frankTheTank72) [reply README.md:29]: Number of SIPs for this type ...

**Comment by frankTheTank72** [comment]
> Good job. Please review my suggestions and replace really _all_ EIPs by SIPs

done

**Review by jjos2372 — APPROVED** [review]

**Pull request merged**